### ISSUE TITLE
Specify the name / path of the images uploaded to S3

### ISSUE THREAD

**Issue by TimPietrusky** [blib-la/runpod-worker-comfy#49]
**Is your feature request related to a problem? Please describe.**

When using the AWS S3 upload, the path where the image is stored and the name of the file can't be configured. These things are generated automatically. 

**Describe the solution you'd like**

I want to actually configure where the image is stored and how it is called. The best thing would be, that it just has the same name as the image in ComfyUI.

**Comment by TimPietrusky** [comment]
We could do this by not using `upload_image` from the RunPod python SDK, but [`upload_file_to_bucket`](https://github.com/runpod/runpod-python/blob/main/docs/serverless/utils/rp_upload.md#upload_file_to_bucket).

**Comment by TimPietruskyRunPod** [comment]
The community is currently implementing this in #191 ("feat: add BUCKET_PREFIX support for S3 uploads"). When \`BUCKET_PREFIX\` is set, files upload to \`{prefix}/{job_id}/{original_filename}\` — preserving the original filename and giving you a configurable prefix. That covers the "specify the name/path of the images uploaded to S3" ask in this issue.

I reviewed the PR today and it's waiting on a changeset entry from the contributor before merge. Tracking via #190.

Closing this in favor of #190.
